### PR TITLE
BJS2-10382 | Stream API 1

### PR DESCRIPTION
**Pull request by DROFEL**

### CODE REVIEW

File: src/main/java/faang/school/godbless/streamapi_1/StreamLib.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package faang.school.godbless.streamapi_1;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+public class StreamLib {
+    public static List<Integer> evenList(List<Integer> list) {
```
Comment (pablo023) [inline src/main/java/faang/school/godbless/streamapi_1/StreamLib.java:9]: You don't need to implement this kind of functionality. The requirement is to create a method which returns sum of all even numbers

File: src/main/java/faang/school/godbless/streamapi_1/StreamLib.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    public static Integer maxNumber(List<Integer> list) {
+        return list.stream().mapToInt(Integer::intValue).max().orElse(0);
+    }
+
+    public static Double averageList(List<Integer> list) {
+        return list.stream().mapToInt(Integer::intValue).average().orElse(0);
+    }
+
+    public static Integer grepCount(List<String> list, String pattern) {
+        return (int) list.stream().filter(s -> s.startsWith("a")).count();
```
Comment (pablo023) [inline src/main/java/faang/school/godbless/streamapi_1/StreamLib.java:22]: Why "a", if you have "pattern" as a parameter?

**Review by pablo023 — CHANGES_REQUESTED** [review]
Check the task plz and fix the logic here

File: src/main/java/faang/school/godbless/streamapi_1/StreamLib.java (hunk 2 above)
Comment (DROFEL) [reply src/main/java/faang/school/godbless/streamapi_1/StreamLib.java]: Oooops :)

File: src/main/java/faang/school/godbless/streamapi_1/StreamLib.java (hunk 1 above)
Comment (DROFEL) [reply src/main/java/faang/school/godbless/streamapi_1/StreamLib.java]: fixed

**Review by pablo023 — APPROVED** [review]

**Pull request merged**